libs/common/src/main/kotlin/grunnlag/opplysningstyper/Opplysningstype.kt

```diff
@@ -38,8 +38,6 @@ enum class Opplysningstype {
     PERSONROLLE,
     UTENLANDSOPPHOLD, // Bruks kun idag for informasjon fra søknaden
     UTENLANDSADRESSE,
-    INNTEKT,
-    ARBEIDSFORHOLD,
```

Comment:
Fjerning av disse kan potensielt skape litt troubles når man henter ut et grunnlag 🤔

Reply:
Ja, var den jeg også var litt bekymret for. Det er jo ikke noe vi har behov for p.t., men kan legge de tilbake om vi ikke tror grunnlag takler det 🤔

Reply:
Det ligger en del data i grunnlagsdatabasen på INNTEKT og ARBEIDSFORHOLD som er fint å få slettet

Reply:
f5a341e288b21699a6ca29dd7646c83920f8e5fe